pkgs/servers/sql/postgresql/pgjwt/default.nix

```diff
@@ -0,0 +1,24 @@
+{stdenv, fetchgit}:
+stdenv.mkDerivation rec {
+  name="pgjwt-${version}";
+  version="0.0.1";
+  src = fetchgit {
```

Comment:
nope, just simple repo.
Sure, I can change that. Does fetchFromGitHub has any advantage over that?